R/Renderer.R

```diff
@@ -140,6 +140,13 @@ Renderer <- R6::R6Class( # nolint: object_name_linter.
       switch(text_style,
         "default" = block_content,
         "verbatim" = paste0("\n```\n", block_content, "\n```\n"),
+        "rcode" =
+          paste(
```

Comment:
Is there a reason for a paste over sprintf?

Reply:
As I remember sprintf is quicker than paste0, and surely sprintf is more elegant for longer text.

```
microbenchmark::microbenchmark(paste0("Hello", " World"), sprintf("Hello %s", "World"))
Unit: nanoseconds
                         expr min  lq   mean median  uq  max neval
    paste0("Hello", " World") 779 820 922.09    820 861 7667   100
 sprintf("Hello %s", "World") 287 328 387.04    328 369 3321   100
```

so the performance difference is minimal.

Reply:
ah I see now, I test here different solutions and this paste should be removed:)
This was the places I fought a lot. 
Good catch.